be/src/vec/exec/format/json/new_json_reader.cpp

```diff
@@ -1657,7 +1657,19 @@ Status NewJsonReader::_simdjson_write_columns_by_jsonpath(
                 return st;
             }
         }
-        if (i >= _parsed_jsonpaths.size() || st.is<NOT_FOUND>()) {
+        if (i < _parsed_jsonpaths.size() && JsonFunctions::is_root_path(_parsed_jsonpaths[i])) {
+            // Indicate that the jsonpath is "$.", read the full root json object, insert the original doc directly
```

Comment:
Does this feature, that load all fields in JSON to variant column, match the need for users? I think usually users only need some fields in JSON, not all fields.

Reply:
some user need this feature， map all columns to a variant column